lib/spoom/cli/bump.rb

```diff
@@ -75,12 +75,14 @@ def bump(directory = ".")
           exit(0)
         end
 
-        Sorbet::Sigils.change_sigil_in_files(files_to_bump, to)
+        update_to = files_to_bump.each_with_object({}) { |path, update_to| update_to[path] = to }
```

Comment:
Isn't `to` always the same value here? Why do we need tho build this hash with it then?